crates/sui-types/src/sui_system_state/sui_system_state_inner_v1.rs

```diff
@@ -466,7 +454,7 @@ impl SuiSystemStateTrait for SuiSystemStateInnerV1 {
         CommitteeWithNetworkMetadata {
             committee: Committee::new(self.epoch, voting_rights)
                 // unwrap is safe because we should have verified the committee on-chain.
-                // TODO: Make sure we actually verify it.
+                // MUSTFIX: Make sure we always have a valid committee
```

Comment:
Looking at when this might fail: https://github.com/MystenLabs/sui/blob/eb13b1bd8105b40d83d4d0fe21262a3e09cf6266/crates/sui-types/src/committee.rs#L49-L66, it seems that it fails when we have 0 members in the committee, or when we have more than max u32 members, or when someone's voting right is 0. All of these cases seem to be impossible given our min stake setup. Can these checks be removed, or we can keep them here since if they are ever hit we are in extremely bad state anyway?

Reply:
The only thing I am not 100% sure is the voting power being 0

Reply:
Well with low stake departure landed every active validator will have at least 20M SUI staked with them, so everyone will have at least 20 voting power points right (since the total is 10000)?

Reply:
That's fair. Let me do that in a separate PR. Committee::new is called at a few dozen places. So it will probably end up touching a lot of files.

Reply:
Remove the checks and make Commit::new non-fallable